docs/dg/dev/integrate-and-configure/integrate-sitemap-generation.md

```diff
@@ -0,0 +1,378 @@
+---
+title: Integrate Sitemap module
+description: Learn the prerequisites and how to enable and integrate Sitemap module into a Spryker based project.
+last_updated: March 4, 2025
+template: howto-guide-template
+---
+
+{% info_block warningBox "" %}
+
+The Sitemap module is currently in beta, but an official release is coming soon. Feel free to explore the available documentation. We'll continue updating it with details on future releases.
```

Comment:
What do you mean by beta? Is this available to customers? How does this affect docs?

Reply:
It's available, and beta means that we released an unstable version 0.*, so we do not promise anything, major changes are possible, etc, until future 1.* releases. I would not say it affects the doc, it's more like an additional warning for the customer who wants to use it.